portfolio-project-matching-app/components/Nav.js

```diff
@@ -1,19 +1,42 @@
+import { useRouter } from 'next/router'
 import Link from 'next/link'
+import Button from './Button'
+import styles from './Nav.module.css'
+import { useEffect, useState } from 'react'
 // import tailwind from 'tailwind/tailwindcss.css'
 
 const Nav = () => {
+    let router = useRouter();
+    const [path, setPath] = useState('');
+    useEffect(
+        () => {
+            setPath(router.pathname);
+        }
+    )
     return (
-        <nav className="mySillyStyle">
-            <ul className="flex">
-                <li className="mr-6">
-                    <Link href='/' >
-                        <a className="text-white text-lg hover:text-blue-100">Home</a>
+        <nav className={styles.nav}>
+            <ul className="grid grid-cols-3">
+                <li className="col-start-2 justify-self-center m-2">
+                    <Link href="/">
+                        <a className="col-start-2 font-medium text-3xl  text-gray-900">Project Finder</a>
```

Comment:
I think Josh did an awesome job on this and look forward to seeing it with our color palette. We could probably use some free online service to punch up our main logo on the navbar. Maybe even create a free favicon.